README.md

```diff
@@ -125,23 +153,94 @@ const view = mathbox
   });
 ```
 
-You can select objects using `.select()` and a CSS-like selector to get a jQuery-like selection:
+Use your mouse to click and drag the camera's orientation, and zoom in and out:
+
+![2023-01-19 11 32
+59](https://user-images.githubusercontent.com/69635/213530497-22cdf2c2-bea6-4ef4-beea-fbebf73c85d4.gif)
+
+Each primitive call:
+
+- creates a new element
+- inserts it into the tree
+- returns a version of the API object with its selection focused on the new element.
+
+Calling `print()` on some selection will print a representation to the console
+of the selection and any children. For example, `view.print()` prints the
+following:
+
+```jsx
+<cartesian
+  range={[
+    [-2, 2],
+    [-1, 1],
+    [-1, 1],
+  ]}
+  scale={[2, 1, 1]}
+>
+  <axis axis={1} />
+  <axis axis={2} />
+</cartesian>
+```
+
+Select objects using `.select()` and a CSS-like selector to get a jQuery-like
+selection:
 
 ```javascript
-mathbox.select("cartesian > axis");
+root.select("cartesian > axis");
 ```
 
-Use `.print()`, `.inspect()` and `.debug()` to show information about a selection.
+Next, visit the [Quick Start](docs/intro.md) page for a more involved example
+that builds up an animating, interactive mathematical graph with labeled axes.
+
+## Docs & Help
+
+For help, see the following resources:
+
+- [Glossary](docs/glossary.md) of terms to help get familiar with MathBox and WebGL.
+- [MathBox API](docs/api.md) for typical usage.
+- [List of Primitives](docs/primitives.md) for a full element reference.
+- [Writing Custom Shaders](docs/shaders.md) for info on custom shaders and GPU-side processing.
+- [Context API](docs/context.md) for advanced usage.
 
----
+For more involved questions, or just to say hi, please join us in the [MathBox
+Google Group][google-group-url].
 
 ## Related Projects
 
 - [threestrap](https://github.com/unconed/threestrap) - Three.js bootstrapper
-- [shadergraph](https://github.com/unconed/shadergraph) - Functional GLSL linker
+- [shadergraph][shadergraph-url] - Functional GLSL linker
+- [MathBox-react](https://github.com/christopherChudzicki/mathbox-react) - React bindings for MathBox
+- [MathBox.cljs](https://github.com/mentat-collective/mathbox.cljs) - ClojureScript bindings for MathBox via MathBox-react
+
+## Who's using MathBox?
 
----
+- [Math3D online graphing calculator](https://www.math3d.org/)
+- [KineticGraphs JS Engine](https://kineticgraphs.org/) ([code](https://github.com/cmakler/kgjs))
+- [Textbook: "Interactive Linear Algebra""](https://textbooks.math.gatech.edu/ila/) ([code](https://github.com/QBobWatson/ila))
+- Many visualizations at [Sam Zhang's homepage](https://sam.zhang.fyi/#projects)
+- Jesse Bettencourt's [Torus Knot Fibration](http://jessebett.com/TorusKnotFibration/) Master's project ([code](https://github.com/jessebett/TorusKnotFibration))
+- [Interactive knot visualizations](https://rockey-math.github.io/mathbox/graph3d-curve)
```

Comment:
Any of your projects you'd like to add here?

Reply:
Once I get them up to snuff I'll add them as I go.